packages/boxed-expression-component/src/expressions/FunctionExpression/FunctionKindSelector.tsx

```diff
@@ -45,6 +44,73 @@ export const FunctionKindSelector: React.FunctionComponent<FunctionKindSelectorP
     [onFunctionKindSelect]
   );
 
+  function iconForFunctionKind(functionKind: FunctionExpressionDefinitionKind): {
+    icon: React.ReactNode;
+    description: String;
+  } {
+    switch (functionKind) {
+      case FunctionExpressionDefinitionKind.Feel:
+        return {
+          icon: (
+            <span
+              style={{
+                fontSize: "0.8em",
+                fontWeight: "bold",
+              }}
+            >
+              FEEL
+            </span>
+          ),
+          description:
+            "Define function as a 'Friendly Enough Expression Language (FEEL)' expression. This is the default.",
+        };
+      case FunctionExpressionDefinitionKind.Java:
+        return {
+          icon: (
+            <span
+              style={{
+                fontSize: "0.8em",
+                fontWeight: "bold",
+              }}
+            >
+              JAVA
+            </span>
+          ),
+          description:
+            "Define full qualified java class name and a public static method signature to invoke.\nThe method-signature consist of the name of the method, followed by an argument list of the argument types.",
```

Comment:
@jomarko Shouldn't we use `i18n` for these descriptions?

Reply:
I wouldn't stress too much with i18n for now. We have the mechanisms but we're not using it everywhere. I understand it's kind of a pain to use `i18n` right now, and we're gonna need to revisit this at some point in the future.